php/map.jinja

```diff
@@ -1,500 +1,3247 @@
+                            'geoip': 'php-geoip',
+                            'geshi': 'php-geshi',
+                            'gettext': 'php' + php_version,
+                            'gmp': 'php' + php_version + '-gmp',
+                            'hhvm': 'hhvm',
+                            'imagick': 'php-imagick',
+                            'imap': 'php' + php_version + '-imap',
+                            'intl': 'php' + php_version + '-intl',
+                            'json': 'php' + php_version + '-json',
+                            'ldap': 'php' + php_version + '-ldap',
+                            'local_bin': '/usr/local/bin',
+                            'auth-sasl': 'php-auth-sasl',
+                            'mail': 'php-mail',
+                            'mbstring': 'php' + php_version + '-mbstring',
+                            'mcrypt': 'php' + php_version + '-mcrypt',
+                            'memcache': 'php-memcache',
+                            'memcached': 'php-memcached',
+                            'mongo': 'php-mongo',
+                            'mongodb': 'php-mongodb',
+                            'mysql': 'php' + php_version + '-mysql',
+                            'mysqlnd': 'php' + php_version + '-mysql',
+                            'net-smtp': 'php-net-smtp',
+                            'net4': 'php-net-ipv4',
+                            'net6': 'php-net-ipv6',
+                            'oauth': 'php-oauth',
+                            'opcache': 'php' + php_version + '-opcache',
+                            'pear': 'php-pear',
+                            'pgsql': 'php' + php_version + '-pgsql',
+                            'php': 'php' + php_version,
+                            'phpenmod_command': 'phpenmod -v' + php_version,
+                            'pspell': 'php' + php_version + '-pspell',
+                            'redis': 'php' + php_version + '-redis',
+                            'seclib': ['php-phpseclib', 'php-seclib'],
+                            'snmp': 'php' + php_version + '-snmp',
+                            'soap': 'php' + php_version + '-soap',
+                            'ssh2': 'php-ssh2',
+                            'sqlite': 'php' + php_version + '-sqlite3',
+                            'suhosin5_ext': 'suhosin.so',
+                            'suhosin5_repo': 'https://github.com/sektioneins/suhosin',
+                            'suhosin7_ext': 'suhosin7.so',
+                            'suhosin7_repo': 'https://github.com/sektioneins/suhosin7',
+                            'sybase': 'php' + php_version + '-sybase',
+                            'tcpdf': 'php-tcpdf',
+                            'temp_dir': '/tmp',
+                            'tidy': 'php' + php_version + '-tidy',
+                            'xdebug': 'php' + php_version + '-xdebug',
+                            'xml': 'php' + php_version + '-xml',
```

Comment:
```suggestion
                            'xml': ['php' + php_version + '-xml', 'php' + php_version + '-xmlrpc'],
```

Reply:
All of these changes are from the merge of #175, which took place after this PR was first started.